packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -76,7 +76,7 @@ import { ResourceWatchersRegistry } from './resource-watchers-registry.js';
 // only the connectivity to the current context will be checked
 const MAX_NON_CURRENT_CONTEXTS_TO_CHECK = 10;
 
-const kindsWithEvents = ['Deployment', 'Pod'];
+const KINDS_WITH_EVENTS: (string | undefined)[] = ['Deployment', 'Pod', 'Node'];
```

Comment:
nitpick/suggestion/non blocking: Using `Array<string | undefined>` might be easier to read.